doc/source/index.rst

```diff
@@ -93,7 +94,7 @@ Ansys Q3D Extractor, Ansys Siwave, and Ansys Icepak using electrical CAD (ECAD)
 Mechanical CAD (MCAD) workflows.
 
 In addition, AEDT includes direct links to the complete Ansys portfolio of thermal, fluid,
-and mechanical solvers for comprehensive multiphysics analysis.
+and Mechanical solvers for comprehensive multiphysics analysis.
```

Comment:
I was told that this was "mechanical" in the general sense and so should be lower case.